R/summary.R

```diff
@@ -332,10 +331,6 @@ summary.gs_design <- function(object,
   # If the method is WLR, change HR to wHR
   if (method == "wlr") xy <- replace_names(xy, c("~hr at bound" = "~whr at bound"))
 
-  # If the method is COMBO, remove the column of "~HR at bound", and remove AHR from header
-  if (method == "combo" && "~hr at bound" %in% names(xy))
-    stop("'~hr at bound' can't be displayed!")
-
```

Comment:
Since `~hr at bound` can't possibly exist in `xy`, perhaps we can remove this check. It doesn't break any test.

@LittleBeannie I'm not sure if this is the right thing to do.

Reply:
Hi @yihui , yes, you are right that `~hr at bound` does not exist in `xy`, because it is not part of the raw output from the MaxCombo test (see screenshot below).

Thanks for the confirmation!
![image](https://github.com/user-attachments/assets/e79654dd-bf13-419d-8634-88a62d45d317)

Reply:
Okay, then I guess we don't really need the check here. Thanks for letting me know!